Catappult/appcoins-wallet-android
# [APPC-4516] IAP error generic error

**What does this PR do?**

   Created the IAP generic error screen.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] ImgError.kt
- [ ] MainFragment.kt
- [ ] IABButtons.kt
- [ ] IABGenericError.kt
- [ ] Palette.kt
- [ ] ic_support_chat_dark.xml

**How should this be manually tested?**

  Start the IAP flow. Check the error screen. Press the bottom button to navigate between the different states.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-4516](https://aptoide.atlassian.net/browse/APPC-4516)

**Code Review Checklist**



[APPC-4516]: https://aptoide.atlassian.net/browse/APPC-4516?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
